Sources/AppcuesKit/Presentation/Debugger/FloatingView/DebugView.swift

```diff
@@ -495,6 +495,6 @@ extension DebugView {
         case open
         case close
         case reposition
-        case screenCapture
+        case screenCapture(Authorization)
```

Comment:
I suspect we're going to want a new `JWT` struct instead of `Authorization` here eventually since we're going to need to decode the JWT to get the expiry date. Obviously this is fine until the need arises.

Reply:
Perhaps, I was hoping to avoid that honestly - do we need to know? What would we do with that? Maybe we can just detect the 403 and give an appropriate error message to say re-establish your session?

Reply:
The main thing I think we talked about (I think when you were away) was automatically hiding the capture button once the token is expired. A minor quality of life improvement versus capturing and getting an error, so definitely not required.

It's quite simple to decode just the value we need without a library or anything (something like [this](https://stackoverflow.com/a/40915703)) so it wouldn't be a huge lift to do.

Reply:
I see, yeah that would be a nice to have. Thanks